ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/json/UdfJsonMapper.java

```diff
@@ -15,26 +15,57 @@
 
 package io.confluent.ksql.function.udf.json;
 
+import com.fasterxml.jackson.core.JacksonException;
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
 import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import io.confluent.ksql.function.KsqlFunctionException;
 
 /**
  * Shared Object mapper used by JSON processing UDFs
  */
-public enum UdfJsonMapper {
```

Comment:
Any *remote* chance that a downstream client could be depending on this?  Is this a public API change?

Reply:
I don't think so. To my knowledge, we do not consider internal classes part of the public API (except for the interfaces explicitly exposed in the configuration).

Reply:
In the case, let's :shipit: !